doc.config.json

```diff
@@ -94,6 +94,16 @@
                 "path": "/docs/contributions.md"
               }
             ]
+          },
+          {
+            "title": "Guides",
+            "disableAutoPrefix": true,
+            "routes": [
+              {
+                "title": "API with Nextjs",
```

Comment:
Change to Next.js and search the doc in case there are others

Reply:
Perhaps the title should be something like "Build a todo list with Nextjs" ?